frontend/src/components/templates/Stats.js

```diff
@@ -74,18 +74,50 @@ export const Stats = () => {
   const [stats, setStats] = useState(null);
   const { targetApi } = useParams();
 
+  async function getTargetAPIList(targetApi) {
+    let result = [];
```

Comment:
Variable initializer is redundant 
```suggestion
    let result;
```